cli/identity_new.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2024 Democratized Data Foundation
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package cli
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/sourcenetwork/defradb/acp/identity"
+)
+
+func MakeIdentityNewCommand() *cobra.Command {
+	var cmd = &cobra.Command{
+		Use:   "new",
+		Short: "Generate a new identity",
+		Long: `Generate a new identity
```

Comment:
suggestion: would be useful for users to have more detailed documentation on why it's needed and how to use it or something like this.

Reply:
Happy to take suggestions on what I should add. As for why it's needed (it's not) it's just a key pair + did generation utility command incase users ever want to generate a new identity using defradb rather than another key pair generation tool.